src/frontend/src/common/routes.ts

```diff
@@ -1,29 +1,30 @@
 export enum ROUTES {
   HOMEPAGE = "/",
```

Comment:
I have always wanted to alphabetize this list

Reply:
godspeed